l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py

```diff
@@ -31,7 +31,7 @@ def test_l10n_nl_xaf_auditfile_export_default(self):
         zf = StringIO(b64decode(self.export.auditfile))
         with ZipFile(zf, 'r') as archive:
             contents = archive.read(archive.namelist()[0])
-        self.assertTrue(contents.startswith('<?xml '))
```

Comment:
Why these changes?

Reply:
I see that the `l10n_nl_xaf_auditfile_export` test is failing on this line, but I cannot see what it generates instead of `<?xml `. And on my local the test is passing. An assertEquals would show me directly in Travis what is generated instead, so that I can fix it.

Reply:
OK, thanks for the explanation!